Jakefile

```diff
@@ -31,6 +31,19 @@ task('minify', function () {
   console.log('Minification completed.');
 });
 
+task('doc', function () {
+  exec('./node_modules/.bin/rimraf out');
```

Comment:
The `exec('rm -rf out');` obvs. won't work on Windows, but I'm not sure how well `browserify`, `uglify`, et al work on Windows anyhow.